README.md

```diff
@@ -2,6 +2,12 @@
 
 In this project we create a virtual network of cassandra instances to benchmark them. The goals is to perform smart request duplications to increase chanses of meeting strict latency guarantees.
 
+Starting the topology can be done by running the following command in the root folder of the repo:
+
+```sh
+> sudo ./pttopology
```

Comment:
Is there a reason for including the `>` character?

Reply:
We could use the '$' sign..?

Reply:
This file has this as the convention.